### PR TITLE
ASCII Collating Sequence image converted into Table

### PR DESCRIPTION
**Pull request by aserputov**
Issue #66.

### CODE REVIEW

**Comment by mkim219** [comment]
@aserputov I wonder how you have done ASCII table. Your way is more clear because you have extra column between the tables to make more distinguishable and readable. I will change to insert empty column to my EBCDIC collating sequence.

**Comment by aserputov** [comment]
@Yoda-Canada true. I will add subtitle now, and the | might be mixed with the table.

**Comment by aserputov** [comment]
@Yoda-Canada could you please send the website link from which you took the screenshot of the subtitle?

**Comment by aserputov** [comment]
I think this PR is good to be merged. What do y'all think?

File: docs/Resources-Appendices/ascii-collating-sequence.md [hunk 1]
```diff
@@ -4,5 +4,50 @@ sidebar_position: 1
...
+| 00011111 | 1F   | 31   | US     |     | 01001010 | 4A   | 74   | J      |     | 01110101 | 75   | 117  | u      |
+| 00100000 | 20   | 32   | SP     |     | 01001011 | 4B   | 75   | K      |     | 01110110 | 76   | 118  | v      |
+| 00100001 | 21   | 33   | !      |     | 01001100 | 4C   | 76   | L      |     | 01110111 | 77   | 119  | w      |
+| 00100010 | 22   | 34   | "      |     | 01001101 | 4D   | 77   | M      |     | 01111000 | 78   | 120  | x      |
+| 00100011 | 23   | 35   | #      |     | 01001110 | 4E   | 78   | N      |     | 01111001 | 79   | 121  | y      |
+| 00100100 | 24   | 36   | $      |     | 01001111 | 4F   | 79   | O      |     | 01111010 | 7A   | 122  | z      |
+| 00100101 | 25   | 37   | %      |     | 01010000 | 50   | 80   | P      |     | 01111011 | 7B   | 123  | {      |
+| 00100110 | 26   | 38   | &      |     | 01010001 | 51   | 81   | Q      |     | 01111100 | 7C   | 124  |<img width="11" alt="Screen Shot 2021-11-15 at 5 51 05 PM" src="https://user-images.githubusercontent.com/65831678/141865226-06c0f43b-a52f-4f7a-8b74-97398a427163.png">      |
```
Comment (humphd) [inline docs/Resources-Appendices/ascii-collating-sequence.md:49]: If you're going to use an image, it needs to live in the repo (where we control it) vs in an external site.

File: package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "name": "ipc144",
   "version": "0.0.0",
   "private": true,
-  "license" : "GFDL-1.3-no-invariants-or-later",
```
Comment (humphd) [inline package.json:5]: Please revert changes to this file, it's not needed here.

**Review by humphd — CHANGES_REQUESTED** [review]
Please revert the changes to `package-lock.json`, since it's unnecessary for this change.

```
git checkout master package-lock.json
git checkout master package.json
```

File: docs/Resources-Appendices/ascii-collating-sequence.md (hunk 1 above)
Comment (aserputov) [reply docs/Resources-Appendices/ascii-collating-sequence.md:49]: I don't have any idea how to add | into the table. Any advice?

File: package.json (hunk 2 above)
Comment (aserputov) [reply package.json:5]: Done with it, could you please check if you like it?

File: docs/Resources-Appendices/ascii-collating-sequence.md (hunk 1 above)
Comment (aserputov) [reply docs/Resources-Appendices/ascii-collating-sequence.md:49]: If you think it supposes to be an image, too, I will redo it now.

**Comment by Yoda-Canada** [comment]
> @Yoda-Canada could you please send the website link from which you took the screenshot of the subtitle?

Sorry, I replied late. the link is https://ict.senecacollege.ca/~ipc144/pages/content/ascii.html

File: docs/Resources-Appendices/ascii-collating-sequence.md (hunk 1 above)
Comment (aserputov) [reply docs/Resources-Appendices/ascii-collating-sequence.md]: Fixed, with \|

File: package.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
   "name": "ipc144",
```
Comment (humphd) [inline package.json:2]: None of the changes to `package.json` should be in this PR.  You can revert them:

```sh
git checkout main package.json
```

**Review by humphd — CHANGES_REQUESTED** [review]
I think this looks great.  I'll get @CameronGray1210 to comment on it tomorrow when we do reviews.

I've left a comment about removing the `package.json` changes.

File: package.json (hunk 3 above)
Comment (aserputov) [reply package.json]: I think now it's correct because it shows that only the file changed, right?

**Review by humphd — COMMENTED** [review]
The changes to the md file are good (no package.json), but you now have 15 commits.  I want to see 1.

**Review by humphd — CHANGES_REQUESTED** [review]
No, you have 17 commits, see https://github.com/Seneca-ICTOER/IPC144/pull/80/commits.  It should be 1.  Please squash and rebase properly before requesting another review.

**Comment by aserputov** [comment]
Done.

**Review by humphd — APPROVED** [review]
I'll file a follow-up issue to change the order of some of this, but this is way better.

**Pull request merged**